cron.sh

```diff
@@ -65,7 +65,10 @@ while true; do
                 branch="${remote#origin/}"
                 git branch --track "$branch" "$remote";
                 git checkout "$branch"
-                git pull
+                if ! git pull; then
+                    git branch -D "$branch"
```

Comment:
IIRC trying to remove a checked out branch will fail. I would say `git reset --hard "$remote"` is more applicable.